lib/package/dynamodb/compileMethodsToDynamodb.js

```diff
@@ -0,0 +1,387 @@
+
+module.exports = {
+  compileMethodsToDynamodb() {
+    this.validated.events.forEach((event) => {
+      if (event.serviceName == 'dynamodb') {
+        event.http.action = this.getDynamodbAction(event.http)
+        const resourceId = this.getResourceId(event.http.path)
+        const resourceName = this.getResourceName(event.http.path)
+
+        const template = {
+          Type: 'AWS::ApiGateway::Method',
+          Properties: {
+            HttpMethod: event.http.method.toUpperCase(),
+            RequestParameters: {},
+            AuthorizationType: event.http.auth.authorizationType,
+            AuthorizationScopes: event.http.auth.authorizationScopes,
+            AuthorizerId: event.http.auth.authorizerId,
+            ApiKeyRequired: Boolean(event.http.private),
+            ResourceId: resourceId,
+            RestApiId: this.provider.getApiGatewayRestApiId()
+          }
+        }
+
+        _.merge(
+          template,
+          this.getDynamodbMethodIntegration(event.http),
+          this.getMethodResponses(event.http)
+        )
+
+        const methodLogicalId = this.provider.naming.getMethodLogicalId(
+          resourceName,
+          event.http.method
+        )
+
+        this.apiGatewayMethodLogicalIds.push(methodLogicalId)
+
+        _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources, {
+          [methodLogicalId]: template
+        })
+      }
+    })
+  },
+
+  getDynamodbAction(http) {
+    if (!_.has(http, 'action')) {
+      switch (http.method.toUpperCase()) {
+        case 'PUT':
+          return 'PutItem'
+        case 'POST':
+          return 'PutItem'
+        case 'GET':
+          return 'GetItem'
+        case 'DELETE':
+          return 'DeleteItem'
+        case 'PATCH':
+          return 'UpdateItem'
+      }
+    }
+
+    return http.action
+  },
+
+  getDynamodbMethodIntegration(http) {
+    const integration = {
+      IntegrationHttpMethod: 'POST',
+      Type: 'AWS',
+      Credentials: {
+        'Fn::GetAtt': ['ApigatewayToDynamodbRole', 'Arn']
+      },
+      Uri: {
+        'Fn::Sub': [
+          'arn:aws:apigateway:${AWS::Region}:dynamodb:action/${action}',
+          { action: http.action }
+        ]
+      },
+      PassthroughBehavior: 'NEVER',
+      RequestTemplates: this.getDynamodbIntegrationRequestTemplates(http)
+    }
+
+    const integrationResponse = {
+      IntegrationResponses: [
+        {
+          StatusCode: 200,
+          SelectionPattern: '2\\d{2}',
+          ResponseParameters: {},
+          ResponseTemplates: this.getDefaultDynamodbResponseTemplates(http)
+        },
+        {
+          StatusCode: 400,
+          SelectionPattern: '4\\d{2}',
+          ResponseParameters: {},
+          ResponseTemplates: {}
+        },
+        {
+          StatusCode: 500,
+          SelectionPattern: '5\\d{2}',
+          ResponseParameters: {},
+          ResponseTemplates: {}
+        }
+      ]
+    }
+
+    if (http && http.cors) {
```

Comment:
you can use the utils method here, e.g: `this.addCors(http, integrationResponse)`